### ISSUE TITLE
Comments on "Running .NET 8.0 isolated functions and Azurite Storage in Docker on a Raspberry Pi 5"

### ISSUE THREAD

**Issue by LocalJoost** [LocalJoost/BlogComments#477]
Original article: https://localjoost.github.io/Running-NET-80-isolated-functions-and-Azurite-Storage-in-Docker-on-a-Raspberry-Pi-5/

**Comment by peSHIr** [comment]
Interesting article. Kudos for keeping doing what you're doing. I'm getting too old to keep up to all this, so I'll just keep working with algorithms and data for now. We'll see how long that'll continue. 😉

**Comment by LocalJoost** [comment]
@peSHIr Thank you. Also, I call BS ;) , you are one of the sharpest minds I ever encountered, you are running a development department all by yourself, and you don't get to do that by 'getting too old'. Only some of us feel the need to write blogs, others don't, that's fine. :)

**Comment by peSHIr** [comment]
Yes, and no. I'm still stuck on Framework 4.8 and PowerShell 3, never
having touched containers, WSL, much of Azure (in practice), still having
source code in Azure DevOps source control, not really having touched
git(hub) either. I'm a dinosaur, "just" working geo/3D related algorithms
and huge amounts of data in "old" tech. 🫣 And that's fine. 😜 That's what
I sort of meant, not the blogging per se. 🤣

Op zo 22 dec 2024, 15:12 schreef Joost van Schaik ***@***.***
